plutus-tx/test/Plugin/Errors/Spec.hs

```diff
@@ -33,6 +41,9 @@ machInt = plc @"machInt" (1::Int)
 negativeInt :: CompiledCode Integer
 negativeInt = plc @"negativeInt" (-1 :: Integer)
 
+caseInt :: CompiledCode (Integer -> Bool)
+caseInt = plc @"caseInt" (\(i::Integer) -> case i of { S# i -> True; _ -> False; } )
```

Comment:
Why do we want to expose these low-level details to the user?

Reply:
We don't - this gives an error, that's why it's in the `Error` folder ;)

The problem is that GHC sometimes generates such pattern matches, and then we have to blow up. But I needed a way to make one explicitly for the test.